src/Sulu/Bundle/AdminBundle/Resources/js/index.js

```diff
@@ -44,6 +44,7 @@ function registerFieldTypes() {
 
     // TODO move to correct bundle or even allow to register somehow via the config request
```

Comment:
I think adding the assignments via the server config is also something we should do in the near future.